FBTweak/FBTweakCategory.h

```diff
@@ -50,4 +61,21 @@
  */
 - (void)removeTweakCollection:(FBTweakCollection *)tweakCollection;
 
+/**
+ @abstract Resets all tweaks in \c tweakCollections to their default value.
+ */
+- (void)reset;
+
+/**
+ @abstract Asynchronously updates \c tweakCollections to the latest value, and calls \c completion
+ when done. Upon error \c error is set, otherwise \c error is \c nil.
```

Comment:
Since `error` isn't an argument to this function I suggest to *teak* ;) the doc a little bit:
`Upon error \c error is set, otherwise \c error is \c nil.` ->
`Error is reported by the means of \c completion's \c error argument, which is set upon an error or \c nil otherwise.`

Reply:
Done